server/helpers/youtube-dl/youtube-dl-wrapper.ts

```diff
@@ -20,6 +22,32 @@ const processOptions = {
   maxBuffer: 1024 * 1024 * 30 // 30MB
 }
 
+class YoutubeDLError extends Error {
+  code: YoutubeDLError.CODE
+  cause?: Error // Property to remove once ES2022 is used
```

Comment:
Prefix your comment with `// TODO`

Reply:
Was copied from https://github.com/Chocobozzz/PeerTube/blob/9a3a23a834679e072c15960903e05e667ef14481/server/lib/video-import.ts#L36

Will add the TODO prefix in both places.